src/components/calcite-select/calcite-select.scss

```diff
@@ -104,23 +96,15 @@ select:disabled {
   @apply items-center
     bg-transparent
     border-0
-    border-r
     border-solid
     border-color-input
-    border-l-0
     text-color-2
     flex
     pointer-events-none
     absolute
-    right-0
     inset-y-0;
-}
-
-.icon-container.calcite--rtl {
-  @apply border-l
-    border-r-0 
-    left-0;
-  right: unset;
+  inset-inline-end: 0;
+  border-inline-width: 0 1px;
```

Comment:
I think these are backwards too? Should we use `theme("borderWidth.default")` here?